src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -1,9 +1,17 @@
-import React, {useEffect, useReducer, useState} from 'react';
+import React, {FC} from 'react';
 import AutocompleteItem from "./AutocompleteItem";
 import classes from './Autocomplete.module.css';
+import {ISearchingResult} from "../../types/types";
 
-const Autocomplete = ({query, results, selectedIndex, updateData}) => {
+interface AutocompleteProps {
```

Comment:
If you named one interface with prefix I (ISearchingResult) (which is good) then lets name all other interfaces also with this prefix